mergin/client_push.py

```diff
@@ -259,12 +259,16 @@ def push_project_cancel(job):
     To be called (from main thread) to cancel a job that has uploads in progress.
     Returns once all background tasks have exited (may block for a bit of time).
     """
-    
+
+    job.mp.log.info("user canceled the push...")
     # set job as cancelled
     job.is_cancelled = True
-
     job.executor.shutdown(wait=True)
 
+    resp_cancel = job.mc.post("/v1/project/push/cancel/%s" % job.transaction_id)
```

Comment:
It would be good to wrap this with try / except ClientError and write to log if there was an error.